common/cards/single-use/ladder.ts

```diff
@@ -44,12 +44,6 @@ class LadderSingleUseCard extends SingleUseCard {
 
 			if (inactiveHermitCard === null || !inactiveHermitCardInfo.row) return
 
-			// Swap ailments
```

Comment:
why can this just be removed?

Reply:
Ailments are attached to card instances, not the row

Reply:
gotcha